stable/airflow/README.md

```diff
@@ -474,6 +474,11 @@ Full and up-to-date documentation can be found in the comments of the `values.ya
 
 ## Upgrading
 
+### To 5.0.0
```

Comment:
ok, I prefer updating to 4.8 (now 4.9 i think), since the podDisruptionBudget wasn't working anymore, this restore the behavior from 4.x.

Even remove this section, it is not needed actually